dipy/reconst/mcsd.py

```diff
@@ -421,8 +421,8 @@ def multi_shell_fiber_response(sh_order, bvals, wm_rf, gm_rf, csf_rf,
     MultiShellResponse
         MultiShellResponse object.
     """
-    NUMPY_1_12_PLUS = LooseVersion(np.__version__) > LooseVersion('1.12.0')
-    rcond_value = None if NUMPY_1_12_PLUS else -1
+    NUMPY_1_13_PLUS = LooseVersion(np.__version__) >= LooseVersion('1.14.0')
```

Comment:
The variable name (numpy1.13) and the value (1.14) do not align. Can you rename it `NUMPY_1_14_PLUS`? Thank you

Reply:
Oops, fixed it!